src/include/storage/write_ahead_log/log_io.h

```diff
@@ -47,6 +47,13 @@ struct PosixIoWrappers {
    */
   static void Close(int fd);
 
+  /**
```

Comment:
Like I mentioned in a comment on the pr, revert back all the logging code. I'll work on that in a separate pr parallel to you.

Reply:
Updated